functorch/_src/aot_autograd.py

```diff
@@ -278,17 +279,74 @@ def aot_dispatch_base(flat_fn, flat_args: List[Tensor], aot_config: AOTConfig):
     if config.debug_graphs:
         print("====== Forward (only) graph ======")
         fw_module.print_readable()
-    with track_graph_compiling("inference"):
-        compiled_fw = aot_config.fw_compiler(fw_module, flat_args)
+
+
+    disable_amp = should_disable_autocast(fw_module, flat_args)
+    context = disable_autocast_manager if disable_amp else nullcontext
+
+    with context():
+        with track_graph_compiling("inference"):
```

Comment:
Nit - instead of nesting context managers, can't we put: `with context(), track_graph_compiling("inference")` to prevent unnecessary nesting?